perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -17,7 +17,7 @@ MOUNT_POINT=gcs
 gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
 chmod +x run_load_test_and_fetch_metrics.sh
 ./run_load_test_and_fetch_metrics.sh
-# ls_metrics test - commenting out ls_metrics test till data is populated correctly.
-#cd "./ls_metrics"
-#chmod +x run_ls_benchmark.sh
-#./run_ls_benchmark.sh
+ ls_metrics test - commenting out ls_metrics test till data is populated correctly.
```

Comment:
ls_metrics test is comment and you cannot uncomment it.

Please refer to the PR which commented out this logic to understand what needs to be uncommented.

Reply:
got it. thanks.
fixed